reinforcement_learning/bindings/cs/rl.net.cli/rl.net.cli.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net462</TargetFramework>
```

Comment:
Are we able to make this .net standard?

Reply:
No, because .Net Standard does not specify executable targets. Our only choices are NetCoreApp and Net462. If we swap it to NetCoreApp, it makes it much harder to run on the command, as it does not generate an executable, unless we use .Net Native to AOT-compile it, and I'd rather not do that.